test/evergreen.yml

```diff
@@ -1593,6 +1593,33 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  # creating bunch of spinlock tests so they can be scheduled in parallel
+  - name: spinlock-gcc-test
+    tags: ["spinlock-test"]
```

Comment:
I don't see some new spinlock tests to be added in the foreseeable future, and think the use of tags is a bit over-kill.